examples/find_all_posts_mentioning_github.rb

```diff
@@ -8,8 +8,14 @@ def initialize(keyword, language: nil)
     end
 
     @query = client.query do |query|
-      query.language     = language
-      query.search_query = "sort-order:asc sort:published #{keyword}"
+      query_parts = [
+        "sort-order:asc",
+        "sort:published",
+      ]
+      query_parts << "lang:#{language}" if language
+      query_parts << keyword
```

Comment:
guess this could be stuffed into the `query_parts` right away?

```ruby
     query_parts = [
        "sort-order:asc",
        "sort:published",
        keyword,
      ]
```

Reply:
Jupp, that should work fine.